src/runtime/storageNG/unified-store.ts

```diff
@@ -182,3 +182,8 @@ export type StoreInfo = {
    */
   readonly includeKey?: string;
 };
+
+export interface StoreRegistry {
```

Comment:
as an alternative, why not add registerStore and unregisterStore to context directly (manifest.ts), since the arc already has a reference to the context it would avoid needing an extra parameter in the constructor. (unless the dependency is also problematic cycle-wise).

Reply:
Eventually the context is planned to be a thing that knows everything about the state of the world, including, of course, all available stores. There was a design doc started on that by Raul, but that's unfinished
I don't know if we want to make intermediate changes to the manifest meantime (that's why i originally put the stores registry in the Runtime...). @shans WDYT?

Reply:
Thanks, this is excellent feedback, here is [the doc](https://docs.google.com/document/d/1PVWeMH3rONy6g8zorBSZUyX_oryF2Ilj7RAa5thGdAE/edit) Maria mentioned.

I put together https://github.com/PolymerLabs/arcs/pull/4351 as another option, my two bullets in the description there are the tradeoffs.

I am on the fence -- it does seem it could be nice to have a minimal interface for store register/unregister, for future flexibility. But the other change is quite small and appears sufficient cycles-fix-wise.